### PR TITLE
Refactor UI/Interface misc to TitleCase

### PR DESCRIPTION
**Pull request by pizza2004**
I included a few in this one again because they're all a bit small.

### CODE REVIEW

**Review by duncanspumpkin — APPROVED** [review]

**Comment by duncanspumpkin** [comment]
Will need a rebase due to the other PRs

File: src/openrct2-ui/interface/ViewportInteraction.cpp [hunk 1]
```diff
@@ -721,7 +721,7 @@ static Peep* viewport_interaction_get_closest_peep(ScreenCoordsXY screenCoords,
  *
  *  rct2: 0x0068A15E
  */
-CoordsXY sub_68A15E(const ScreenCoordsXY& screenCoords)
+CoordsXY ViewportInteractionGetTileStartAtCursor(const ScreenCoordsXY& screenCoords)
```
Comment (pizza2004) [inline src/openrct2-ui/interface/ViewportInteraction.cpp:724]: While I was trying to figure out what this function did in order to rename it, I found that removing lines 744 to 763 and simply returning the "initialPos" variable did not seem to change the functionality at all in game.  I tested just about every use case except placing an entrance, but I'm not sure what those lines are doing so I'm not sure how good my tests were.  Might be a place to remove unnecessary code, but I figured it wasn't the scope of this PR.

Comment (duncanspumpkin) [reply src/openrct2-ui/interface/ViewportInteraction.cpp:724]: Yes please do not do any code changes during these refactors no matter how tempting they might be. Make a follow on pr or issue so we can track it.

Comment (duncanspumpkin) [reply src/openrct2-ui/interface/ViewportInteraction.cpp:724]: That bit of code i think is hit when there is columns of land. I think it is required

**Pull request merged**